### PR TITLE
Creating SubBalance controller, service and doc

### PR DESCRIPTION
**Pull request by CNDC-lab**

### CODE REVIEW

**Review by MaksymilianK — CHANGES_REQUESTED** [review]

File: planner-api/openapi.yaml [hunk 1]
```diff
@@ -78,84 +78,66 @@ paths:
...
-        "204":
-          description: "SubBalances Created"
-        "401":
-          description: "User unauthorized"
-
-    get:
-      summary: "List user subBalances"
+              "$ref": "#/components/schemas/SubBalanceInfo"
       responses:
         "200":
```
Comment (MaksymilianK) [inline planner-api/openapi.yaml:92]: 200 -> 201

**Review by MaksymilianK — CHANGES_REQUESTED** [review]

File: planner-api/openapi.yaml [hunk 2]
```diff
@@ -117,7 +117,7 @@ paths:
             schema:
               "$ref": "#/components/schemas/SubBalanceInfo"
       responses:
-        "200":
+        "201":
```
Comment (MaksymilianK) [inline planner-api/openapi.yaml:120]: 201 -> 200 (we return 201 CREATED only when a new resource is created)

**Review by MaksymilianK — CHANGES_REQUESTED** [review]

File: planner-api/openapi.yaml [hunk 3]
```diff
@@ -78,6 +78,69 @@ paths:
...
+      responses:
+        "204":
+          description: "Deleted subBalance"
+        "401":
+          description: "User unauthorized"
+
+  "/api/balance/sub/{id}":
+    get:
+      summary: "Get single subBalance"
+      requestBody:
```
Comment (MaksymilianK) [inline planner-api/openapi.yaml:112]: Why is there a requestBody? It's a GET method.

**Review by MaksymilianK — CHANGES_REQUESTED** [review]

File: planner-api/openapi.yaml (hunk 3 above)
Comment (CNDC-lab) [reply planner-api/openapi.yaml:112]: Sorry for that little bit to much copy/paste work

**Review by MaksymilianK — APPROVED** [review]

**Pull request merged**